src/main/java/org/jenkinsci/test/acceptance/po/Login.java

```diff
@@ -28,7 +31,8 @@ public Login(Jenkins jenkins) {
     public Login doLogin(String user, String password){
         cUser.set(user);
         cPassword.set(password);
-        cLogin.click();
```

Comment:
there are likely many other page object doing the same thing - but this was obvious causing internal tests to reliably fail falsely when testing unhappy paths.

It could probably be worth while sweeping through the code and looking for any "Submit" buttons (not apply) and changing them over, however I've not seen others reliably fail so keeping this change sufficiently focused.

It could also be worthwhile looking at why submit() is not working correctly, but I have spent a few days tracking this down and Selenium is not my best friend right now and this works.